ground-core/docker/config.yml

```diff
@@ -0,0 +1,17 @@
+dbType: neo4j
```

Comment:
Is it possible to not have to duplicate the config file? Or is there a reason we'd explicitly want to do this?